linux/src/sync_linux.c

```diff
@@ -7,10 +7,10 @@
 #include <time.h>
 #include <string.h>
 #include <errno.h>
+#include <syscall.h>

```

Comment:
Cannot really find syscall in C11, this is a Linux thing, so it should be where it was, next to the linux one in line 14 me thinks.

Aaaaand when I look at it : [https://man7.org/linux/man-pages/man2/syscall.2.html](https://man7.org/linux/man-pages/man2/syscall.2.html)

it seems to be used with the sys/ "namespacing"

What prompted this change?

Reply:
I was having build problems, and syscall was one of the things that was giving me trouble. Now that I've got everything else resolved, this change doesn't look needed.